source/catalog.html

```diff
@@ -0,0 +1,296 @@
+                <td>1000 Р.</td>
+              </tr>
+            </table>
+            <a class="button catalog__button">заказать</a>
+          </article>
+          <article class="catalog__item">
+            <a class="catalog__link-img" href="#">
+              <img class="catalog__img" width="68" height="78" src="img/catalog-products/catalog-5.png" alt="Синяя упаковка cat energy 500 грамм со вкусом гречки">
+            </a>
+            <a class="catalog__link" href="#">
+              <h3>Cat Energy SLIM 500 г</h3>
+            </a>
+            <table class="catalog__details">
+              <tr>
+                <td>Масса</td>
+                <td>500 г</td>
+              </tr>
+              <tr>
+                <td>Вкус</td>
+                <td>Гречка</td>
+              </tr>
+              <tr>
+                <td>Цена</td>
+                <td>400 Р.</td>
+              </tr>
+            </table>
+            <a class="button catalog__button">заказать</a>
+          </article>
+          <article class="catalog__item">
+            <a class="catalog__link-img" href="#">
+              <img class="catalog__img" width="84" height="100" src="img/catalog-products/catalog-6.png" alt="Синяя упаковка cat energy 1000 грамм со вкусом гречки">
+            </a>
+            <a class="catalog__link" href="#">
+              <h3>Cat Energy SLIM 1000 г</h3>
+            </a>
+            <table class="catalog__details">
+              <tr>
+                <td>Масса</td>
+                <td>1000 г</td>
+              </tr>
+              <tr>
+                <td>Вкус</td>
+                <td>Гречка</td>
+              </tr>
+              <tr>
+                <td>Цена</td>
+                <td>700 Р.</td>
+              </tr>
+            </table>
+            <a class="button catalog__button">заказать</a>
+          </article>
+          <article class="catalog__item">
+            <a class="catalog__link-img" href="#">
+              <img class="catalog__img" width="72" height="84" src="img/catalog-products/catalog-7.png" alt="Фиолетовая упаковка cat energy 500 грамм со вкусом риса">
+            </a>
+            <a class="catalog__link" href="#">
+              <h3>Cat Energy SLIM 500 г</h3>
+            </a>
+            <table class="catalog__details">
+              <tr>
+                <td>Масса</td>
+                <td>500 г</td>
+              </tr>
+              <tr>
+                <td>Вкус</td>
+                <td>Рис</td>
+              </tr>
+              <tr>
+                <td>Цена</td>
+                <td>500 Р.</td>
+              </tr>
+            </table>
+            <a class="button catalog__button">заказать</a>
+          </article>
+          <div class="catalog__item">
+            <a class="catalog__more-icon" href="#">
+              <svg width="30" height="30" fill="none" xmlns="http://www.w3.org/2000/svg"><path stroke="#D3D3D3" stroke-width="2" d="M15 0v30M30 15H0"/></svg>
```

Comment:
в разметке это лишние атрибуты:
```suggestion
              <svg width="30" height="30" fill="none"><path stroke="#D3D3D3" stroke-width="2" d="M15 0v30M30 15H0"/></svg>
```